test/pass/version-rocmversion-env.cmake

```diff
@@ -0,0 +1,18 @@
+################################################################################
+# Copyright (C) 2017 Advanced Micro Devices, Inc.
+################################################################################
+
+find_program(GIT NAMES git)
+
+set(ENV{ROCM_LIBPATCH_VERSION} "21001")
+file(MAKE_DIRECTORY ${TMP_DIR}/repo)
+test_exec(COMMAND ${GIT} init WORKING_DIRECTORY ${TMP_DIR}/repo)
+write_version_cmake(${TMP_DIR}/repo 1.0 "
```

Comment:
There should be another test like this but we should set the version to `1.0.0` instead of `1.0`.

Reply:
Added the test case version-patch.cmake